### PR TITLE
Closes #1 Add user route and template

### LINKED ISSUE #1: Create User Route
As a user, I should be able to visit '/users' and see a list of current users.

### PR DESCRIPTION
**Pull request by Tman22**
This adds a bunch of cool neat stuff about users.

### CODE REVIEW

File: app/templates/users.hbs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<h1>Hey this is cool {{model}}</h1>
+
+{{outlet}}
```
Comment (martensonbj) [inline app/templates/users.hbs:3]: We don't need this outlet here.

File: tests/unit/routes/users-test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:users', 'Unit | Route | users', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
```
Comment (martensonbj) [inline tests/unit/routes/users-test.js:5]: Comments!! wtf Taylor.

**Review by martensonbj — CHANGES_REQUESTED** [review]
Few things to change.

File: app/templates/users.hbs (hunk 1 above)
Comment (Tman22) [reply app/templates/users.hbs:3]: Soon. Very soon we will. Future thinking

File: tests/unit/routes/users-test.js (hunk 2 above)
Comment (Tman22) [reply tests/unit/routes/users-test.js:5]: @martensonbj Yo this is done!  👍

**Review by martensonbj — APPROVED** [review]

**Pull request merged**